<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog box simple creator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>15</y>
      <width>485</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>45</y>
      <width>485</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dialog_class_name">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>205</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dialog class name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_dialog">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create dialog</string>
    </property>
   </widget>
   <widget class="QLabel" name="dir_with_source_code">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dir with source code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New field no validation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New string field</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New float field</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New int field</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionPolski"/>
    </widget>
    <widget class="QMenu" name="menuDefault_checking">
     <property name="title">
      <string>Default settings</string>
     </property>
     <addaction name="actionCentering_a_string_horizontally"/>
     <addaction name="actionCentering_a_string_vertically"/>
     <addaction name="action_No_change_window_size"/>
     <addaction name="actionWithout_a_question_mark"/>
     <addaction name="actionRemove_unnecessary_spaces"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuDefault_checking"/>
    <addaction name="actionAbout_program"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPolski">
   <property name="text">
    <string>polski</string>
   </property>
  </action>
  <action name="actionCentering_a_string_horizontally">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Centering a string horizontally</string>
   </property>
  </action>
  <action name="actionCentering_a_string_vertically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Centering a string vertically</string>
   </property>
  </action>
  <action name="action_No_change_window_size">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No change window size</string>
   </property>
  </action>
  <action name="actionWithout_a_question_mark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Without a question mark</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemove_unnecessary_spaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove unnecessary spaces</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
